http/http-advanced-reactive/src/test/java/io/quarkus/ts/http/advanced/reactive/HeadersIT.java

```diff
@@ -92,6 +95,22 @@ void testPathSpecificHeaderRulesOrder() {
         cacheControlMatches(response, "max-age=1");
     }
 
+    /*
+     * This test is related to the backport https://github.com/quarkusio/quarkus/pull/41411 , fails with
+     * -Dquarkus.platform.version=3.11.3
+     */
+    @Test
+    void testWithNoAcceptHeader() {
+        final var header = new Header("Accept", null);
+        given()
+                .when()
+                .header(header)
+                .get("/headers")
+                .then()
+                .statusCode(200)
+                .body(CoreMatchers.is("Headers response: ok headers"));
```

Comment:
You don't need `CoreMatchers.is` the `is` is enough as we import `org.hamcrest.Matchers.is`